src/Mailtrap.Abstractions/ContactImports/Models/ContactsImport.cs

```diff
@@ -0,0 +1,63 @@
+namespace Mailtrap.ContactImports.Models;
+
+/// <summary>
+/// Generic response object for contact imports operations.
+/// </summary>
+public record ContactsImport
+{
+    /// <summary>
+    /// Gets created contact imports identifier.
+    /// </summary>
+    ///
+    /// <value>
+    /// Contact imports identifier.
+    /// </value>
+    [JsonPropertyName("id")]
+    [JsonPropertyOrder(1)]
+    [JsonRequired]
+    public long Id { get; set; }
+
+    /// <summary>
+    /// Gets contact imports status.
+    /// </summary>
+    ///
+    /// <value>
+    /// Contact imports status.
+    /// </value>
+    [JsonPropertyName("status")]
+    [JsonPropertyOrder(2)]
+    public ContactsImportStatus? Status { get; set; }
```

Comment:
IMHO status shouldn't be nullable since it is always present and has one of the values.